js_test.sh

```diff
@@ -49,6 +49,8 @@ if [[ $(
     cat "$TMP_FILE" |
     grep -v 'ignored, nothing could be mapped' |
     grep -v "This browser doesn't support the \`onScroll\` event" |
+    grep -v "Warning: Accessing PropTypes via the main React package is deprecated" |
```

Comment:
yeah, I think it's react-telephone-input iirc. I'm subscribed to the issue on their repo for upgrading to fix the error, so when I see that get closed I'll upgrade the dependency and we can remove this line